source/packages/services/installer/src/commands/modules/service/certificateVendor.ts

```diff
@@ -51,16 +55,322 @@ export class CertificateVendorInstaller implements ServiceModule {
         this.assetLibraryStackName = `cdf-assetlibrary-${environment}`;
         this.commandAndControlStackName = `cdf-commandandcontrol-${environment}`;
     }
+    
+    if (updatedAnswers.certificateVendor === undefined) {
+      updatedAnswers.certificateVendor = {};
+    }
+    
+    const pcaAliases = await this.getPcaAliases(answers);
+    updatedAnswers.certificateVendor.pcaAliases = pcaAliases;
 
-    public async prompts(answers: Answers): Promise<Answers> {
```

Comment:
There's a problem with this merge that results in this function getting removed.

Reply:
will fix